catalyst_voices/uikit_example/pubspec.yaml

```diff
@@ -15,6 +15,7 @@ dependencies:
     path: ../packages/catalyst_voices_brands
   catalyst_voices_localization:
     path: ../packages/catalyst_voices_localization
+  collection: ^1.18.0
```

Comment:
Pls make sure to update this dependency version in melos.yaml, then run `melos bs` to sync it everywhere